src/Config.ts

```diff
@@ -1 +1 @@
-export const apiAddress = window.location.origin + "/api"
+export const apiAddress = "https://api.brigade2.io"
```

Comment:
This change was accidental. It needs to be removed before this PR can be merged.